validator/accounts/v2/wallet.go

```diff
@@ -60,6 +49,7 @@ var (
 // and providing secure access to eth2 secrets depending on an
 // associated keymanager (either direct, derived, or remote signing enabled).
 type Wallet struct {
+	walletDir      string
 	accountsPath   string
 	passwordsDir   string
```

Comment:
Whats the difference between walletDir and accountsPath?

Reply:
Yeah, the wallet path is the path of the prysm-wallet-v2, and the accounts path is deeper within that directory